classes/class-twenty-twenty-one-customize.php

```diff
@@ -81,6 +81,40 @@ public function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Add excerpt or full text selector to customizer
+			 */
+
+			$wp_customize->add_section(
+				'theme_settings',
+				array(
+					'title'    => __( 'Theme settings', 'twentytwentyone' ),
+					'priority' => 30,
+				)
+			);
+
+
+			$wp_customize->add_setting(
+				'display_excerpt_or_fullpost',
+				array(
+					'capability' => 'edit_theme_options',
+					'default'    => 'excerpt',
+				)
+			);
+
+			$wp_customize->add_control(
+				'display_excerpt_or_fullpost',
+				array(
+					'type'    => 'radio',
+					'section' => 'theme_settings',
+					'label'   => __( 'On the Posts page, post show:', 'twentytwentyone' ),
```

Comment:
`esc_html__` instead of `__` - same as line 91

```suggestion
					'label'   => esc_html__( 'On the Posts page, post show:', 'twentytwentyone' ),
```